### DISCUSSION TITLE
Webhooks logs is not implemented and is [Sep 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6357] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **arnaldoperez**  
#### that's a great thing to know, you should fix  then this message  
![image0](https://cdn.discordapp.com/attachments/1137112685562052618/1137113480395239464/image.png)  
#  
> 🙋‍♂️ **arnaldoperez**  
#### Something like "Coming soon..." would be more accurate  
#  
> 🙋‍♂️ **arnaldoperez**  
#### is there a guide or something like that to build webhooks for it? I'm having troubles parsing the data incoming from nocodb  
#  
> 🙋‍♂️ **arnaldoperez**  
#### it seems like it's being split in diffrent parts  
![image0](https://cdn.discordapp.com/attachments/1137112685562052618/1137114063277654116/image.png)  
#  
> 🙋‍♂️ **arnaldoperez**  
#### I'm using express and it's native middleware to create this hook  
#  
> 💁 **o1lab**  
#### Thank you for providing the screenshot. We will fix this.  
#  
> 💁 **o1lab**  
#### this shud help   
#### https://docs.nocodb.com/developer-resources/webhooks  
#  
> 🙋‍♂️ **arnaldoperez**  
#### I've been through your docs and I can't find anything helpful related to the development of the hook. Like I just find out that it sends x-www-form-urlencoded because I logged the headers on my api and made a test, but that information is not in the docs  
#  
> 🙋‍♂️ **arnaldoperez**  
#### and as you can see, looks like the url parsing is not working as it should, or maybe I'm doing something wrong, the thing is that there are no docs in that regard  
#  
> 🙋‍♂️ **arnaldoperez**  
#### If I manage to get this webhook working I could share my repo as an sample of a webhook api with express, to fill this gap  
#  
> 👨‍🚀 **wingkwong**  
#### I think the call log has been implemented. by default, u can set `NC_AUTOMATION_LOG_LEVEL` to `ERROR` to show the errors only or `ALL` to show the all messages. The logs are only available for the calls after turning on.  
#  
> 👨‍🚀 **wingkwong**  
#### for parsing the data, i need more info. Can you DM me to share more info on your RunNeApi.js?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1137112685562052618)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
